src/activities/menu/Menu.qml

```diff
@@ -504,9 +675,11 @@ ActivityBase {
             dialogActivityConfig.configItem.save();
         }
         onClose: {
-            ActivityInfoTree.filterByTag(menuActivity.currentTag)
-            ActivityInfoTree.filterLockedActivities()
-            ActivityInfoTree.filterEnabledActivities()
+            if(menuActivity.currentTag != "search"){
+                ActivityInfoTree.filterByTag(menuActivity.currentTag)
+                ActivityInfoTree.filterLockedActivities()
+                ActivityInfoTree.filterEnabledActivities()
+            }
```

Comment:
Here you should call ActivityInfoTree.filterBySearch(input.text) in the else case. Because if the user changes the conf while in the search, the changes won't be reflected